README.md

```diff
@@ -25,6 +25,20 @@ To use viur-cli in your ViUR projects, use [pipenv](https://pypi.org/project/pip
 $ pipenv install --dev viur-cli
 ```
 
+## Packaged scripts
+
+In order to use some of the packaged scripts, it is possible to simply run them from the environment that you installed viur-cli into. So if you are using a pipenv, then the scripts will be available in the shell just like you would run viur-cli by calling viur as well.
+
+A handy script that has to be started this way is viur-2to3, which will automatically crawl through a project folder and detect deprecated syntax. 
+It is not only useful for porting viur2 projects to viur3, but also when there are deprecations or breaking changes in viur3!
+You should start it like this in your project folder:
+
+```sh
+$ viur-2to3 -d .
+```
+
+This will dry-run the script in the current directory and not make any changes, only suggestions. If you want to make the changes, leave out the -d argument and if you are a daring go-getter and like to live dangerously, replace the -d with -x, which will write the suggested changes without making a backup of the changed files.
```

Comment:
```suggestion
This will dry-run the script in the current directory and not make any changes, only suggestions. If you want to make the changes, leave out the `-d` argument and if you are a daring go-getter and like to live dangerously, replace the `-d` with `-x`, which will write the suggested changes without making a backup of the changed files.
```